src/tools/wasm-ctor-eval.cpp

```diff
@@ -489,27 +489,38 @@ struct CtorEvalExternalInterface : EvallingModuleInstance::ExternalInterface {
   }
 };
 
+struct EvalCtorOutcome {
+  // Whether we completely evalled the function (that is, we did not fail, and
+  // we did not only partially eval it).
+  bool evalledCompletely;
+
+  // If the function was evalled completely, and it returns something, that
+  // value is given here.
+  Literals results;
+
+  static EvalCtorOutcome incomplete() { return {false, Literals()}; }
+
+  static EvalCtorOutcome complete(Literals results = Literals()) {
```

Comment:
Looks like nobody is using the default value?  Remove it?

Reply:
Sure, removed.